client/a8c-for-agencies/sections/licenses/licenses-overview/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { Button } from '@automattic/components';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import Layout from 'calypso/a8c-for-agencies/components/layout';
+import LayoutBody from 'calypso/a8c-for-agencies/components/layout/body';
+import LayoutHeader, {
+	LayoutHeaderActions as Actions,
+	LayoutHeaderTitle as Title,
+} from 'calypso/a8c-for-agencies/components/layout/header';
+import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
+import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import LicenseSearch from '../license-search';
+import LicenseStateFilter from '../license-state-filter';
+import LicensesOverviewContext from './context';
+import type {
+	LicenseFilter,
+	LicenseSortDirection,
+	LicenseSortField,
+} from 'calypso/jetpack-cloud/sections/partner-portal/types';
+
+interface Props {
+	filter: LicenseFilter;
+	search: string;
+	currentPage: number;
+	sortField: LicenseSortField;
+	sortDirection: LicenseSortDirection;
+}
+
+export default function LicensesOverview( {
+	filter,
+	search,
+	currentPage,
+	sortDirection,
+	sortField,
+}: Props ) {
+	const translate = useTranslate();
+
+	const title = translate( 'Licenses' );
+
+	const context = {
+		filter,
+		search,
+		currentPage,
+		sortDirection,
+		sortField,
+	};
+
+	const partnerCanIssueLicense = true; // FIXME: get this from state
+
+	const onIssueNewLicenseClick = () => {
+		// TODO: dispatch action to open issue license modal
+	};
+
+	const showEmptyStateContent = false; // FIXME: get this from state
+
+	return (
+		<Layout
+			className={ classNames( 'issue-license' ) }
+			title={ title }
+			wide
+			withBorder
+			sidebarNavigation={ <MobileSidebarNavigation /> }
+		>
+			<PageViewTracker
+				title="Partner Portal > Licenses"
+				path="/partner-portal/licenses/:filter"
+				properties={ { filter } }
+			/>
+			{ /*  TODO: <FETCH_LICENSES_HERE /> */ }
+			<LicensesOverviewContext.Provider value={ context }>
+				<LayoutTop>
+					<LayoutHeader>
+						<Title>{ title } </Title>
+						<Actions>
+							<div className="issue-license__controls">
+								<div className="issue-license__actions">
+									{ /* TODO: <SHOW_PARTNER_KEY_SELECTION_HERE /> */ }
+									<Button
+										disabled={ ! partnerCanIssueLicense }
+										href={ partnerCanIssueLicense ? '/marketplace' : undefined }
+										onClick={ onIssueNewLicenseClick }
+										primary
+										style={ { marginLeft: 'auto' } }
+									>
+										{ translate( 'Issue New License' ) }
```

Comment:
Tiny Nit: just something I quickly noticed, so forgive me for the random comment! Should this be sentence case as per #38231? 

```suggestion
										{ translate( 'Issue new license' ) }
```

Reply:
Thanks for brining this to our notice, @Aurorum!

@jeffgolenski & @keoshi: Should we change it now?

Reply:
I will merge this PR as this PR deals with porting over the changes from Jetpack Cloud to Automattic for Agencies. We will address this in a different PR if any changes are needed.

CC: @Aurorum